app/forms/teacher_interface/alternative_name_form.rb

```diff
@@ -9,6 +9,8 @@ class TeacherInterface::AlternativeNameForm
   attribute :alternative_family_name, :string
 
   validates :application_form, presence: true
+  validates :alternative_given_names, presence: true, if: :has_alternative_name
+  validates :alternative_family_name, presence: true, if: :has_alternative_name
 
```

Comment:
I think we're missing a validation here on `has_alternative_name` to ensure it's either true or false.